src/components/Todo.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="todo">
+    <template class="todo-intarface">
+      <h1
+        :class="{
+          red: checkCountOfDoneTask() < 0.5,
+          yellow: checkCountOfDoneTask() >= 0.5 && checkCountOfDoneTask() < 1,
+          green: checkCountOfDoneTask() === 1,
+        }"
+      >
+        TODO LIST
+      </h1>
+      <input
+        class="todo-input"
+        v-on:keydown.enter="addTodo"
+        type="text"
+        v-model="todo"
+      />
+      <button class="todo-btn" @click="addTodo">add new task</button>
+    </template>
+    <ul class="todo-list">
+      <li
+        v-for="(t, idx) of todos"
+        :key="idx"
+        class="todo-item"
+        @click.self="removeTodo(idx)"
+      >
+        <input
+          class="todo-checkbox"
+          @change.self="changeStatus(t)"
```

Comment:
Usually, simple methods like this can be done inline, i.e. `task.status = !task.status`.
Not everything needs to be a method.

___

Additionally, don't name your variables `a`, `b`, `v` or other letters.
We aren't saving keystrokes 👉 it's better to be as explicit as possible. Use `task` and `index` instead of `t` and `idx`.